tests/unittests/CoreGraphics/CGBitmapContextTests.mm

```diff
@@ -168,3 +181,20 @@ void _TestPixelFormat(const CGRect bounds, const CGBitmapInfo info, DWORD expect
                      0xffb15927 // expectedfg2
                      );
 }
+
+TEST(CGBitmapContext, Contexts) {
+    CGColorSpaceRef rgbColorSpace = CGColorSpaceCreateDeviceRGB();
+
+    CGContextRef context = CGBitmapContextCreate(nullptr, 120, 20, 8, 480, rgbColorSpace, 0);
+
+    EXPECT_EQ(CGBitmapContextGetBitsPerComponent(context), 8);
+    EXPECT_EQ(CGBitmapContextGetBitsPerPixel(context), 32);
+    EXPECT_EQ(CGBitmapContextGetWidth(context), 120);
+    EXPECT_EQ(CGBitmapContextGetHeight(context), 20);
+
+    EXPECT_EQ(CGBitmapContextGetBytesPerRow(context), 480);
+    EXPECT_NE(CGBitmapContextGetData(context), nullptr);
+
+    CGColorSpaceRelease(rgbColorSpace);
+    CGContextRelease(context);
+}
```

Comment:
I might suggest having a test that draws a red rectangle into a 1x1 PRGBA context and checks that the buffer contains `0xff, 0x00, 0x00, 0xff`.

Additionally, you might want to add a test that goes through the image pipeline (Context -> Render -> Create Image -> Read Pixel); that would have caught the GetResolution issue.